Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10.15-alpine
+
+RUN apk add --no-cache --virtual build-dependencies bash git python make g++ ca-certificates
+
+COPY package.json .
+RUN yarn && yarn global add truffle && yarn cache clean
+
+COPY . .
+
+CMD ["sh", "-c"]
```

Comment:
I think this should be your ENTRYPOINT (https://goinbigdata.com/docker-run-vs-cmd-vs-entrypoint/). CMD will be overridden by whatever you run the image with. Also maybe use bash since it's more powerful than sh and we already installed it.

Reply:
Sorry I missed this. New PR with this change #132